include/deal.II/particles/particle.h

```diff
@@ -486,6 +488,91 @@ namespace Particles
     if (n_properties > 0)
       ar &boost::serialization::make_array(properties, n_properties);
   }
+
+
+
+  template <int dim, int spacedim>
+  void
+  Particle<dim, spacedim>::set_location(const Point<spacedim> &new_loc)
```

Comment:
While not technically necessary due to the template, I wanted to hear opinions whether I should add the `inline` keyword here and in all other places. I think we most often add the keyword, but I am not sure whether we aim to provide it everywhere.